tools/i18nExtractKeys.vite.js

```diff
@@ -0,0 +1,38 @@
+import path from 'path';
+
+const uniqueKeys = new Set();
+
+function extractRegexPlugin({regex, extraKeys, fileOutput}) {
+	fileOutput =
```

Comment:
Seems that ??= does not exist in JS.  But ||= is useful..  did not know that JS already has that as well..

Yeah, maybe letting regex and fileOutput being able to override is more harmful than useful :-). Will let only extraKeys coming from outside

Reply:
Ah, it's working already, see: `window.x ??= alert()`.
But since HTML became a "living standard", all this frontend stuff started to evolve way too fast for me, there are so many new CSS properties that I don't know 🤪